compare-two-strings.js

```diff
@@ -1,10 +1,3 @@
-/** 
- * @param {string} str1 is to be raise
- * @param {string} str2 is to be raise
- * @return {boolean} returns true if length of strings is equal
-   otherwise return false
-*/
-
 function compareStrings(str1, str2) {

```

Comment:
/** 
 * compare two strings
 * @param {string} str1 - str1 must be a string or string literal
 * @param {string} str2 - str2 must be a string or string literal
 * @returns {boolean} - length of the strings is equal or not
*/